integrations/docker/images/stage-2/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -304,12 +308,172 @@ function install_tizen_sdk() {
+        'cynara-devel-*.aarch64.rpm'
+        'dbus-1*.aarch64.rpm'
+        'dbus-devel-*.aarch64.rpm'
+        'dbus-libs-1*.aarch64.rpm'
+        'glib2-devel-2*.aarch64.rpm'
+        'hal-api-common-*.aarch64.rpm'
+        'hal-api-sensor-*.aarch64.rpm'
+        'json-glib-devel-*.aarch64.rpm'
+        'libcynara-client-*.aarch64.rpm'
+        'libcynara-commons-*.aarch64.rpm'
+        'libdns_sd-*.aarch64.rpm'
+        'libjson-glib-*.aarch64.rpm'
+        'libnsd-dns-sd-*.aarch64.rpm'
+        'libsessiond-0*.aarch64.rpm'
+        'libsystemd-*.aarch64.rpm'
+        'libtzplatform-config-*.aarch64.rpm'
+        'parcel-0*.aarch64.rpm'
+        'parcel-devel-*.aarch64.rpm'
+        'pkgmgr-info-*.aarch64.rpm'
+        'sensord-*.aarch64.rpm'
+        'sensord-devel-*.aarch64.rpm'
+        'sensord-dummy-*.aarch64.rpm'
+        'vconf-compat-*.aarch64.rpm'
+        'vconf-internal-keys-devel-*.aarch64.rpm'
+    )
+    download "$URL" "${PKG_ARR[@]}"
+
+    # Unified packages (snapshots)
+    URL="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/latest/repos/standard/packages/aarch64/"
+    PKG_ARR=()
+    download "$URL" "${PKG_ARR[@]}"
+
+    # Tizen Developer Platform Certificate
+    URL="http://download.tizen.org/sdk/extensions/Tizen_IoT_Headless/binary/"
+    # Tizen site does not have this package available in version 8.0.
+    # Certificates are the same for 7.0 and 8.0, though.
+    PKG_ARR=(
+        "7.0-iot-things-add-ons_*_ubuntu-64.zip")
+    download "$URL" "${PKG_ARR[@]}"
+
+    # Install all
+    info "Installing Tizen SDK..."
+
+    unzip -o '*.zip'
+    cp -rf data/* "$TIZEN_SDK_ROOT"
+
+    unrpm *.rpm
+    cp -rf lib64 usr "$TIZEN_SDK_SYSROOT"
+
+    # Install secret tool or not
+    if ("$SECRET_TOOL"); then
+        info "Overriding secret tool..."
+        install "$SCRIPT_DIR/secret-tool.py" "$TIZEN_SDK_ROOT/tools/certificate-encryptor/secret-tool"
+    fi
+
+    # Configure Tizen CLI
+    echo "TIZEN_SDK_INSTALLED_PATH=$TIZEN_SDK_ROOT" >"$TIZEN_SDK_ROOT/sdk.info"
+    echo "TIZEN_SDK_DATA_PATH=$TIZEN_SDK_DATA_PATH" >>"$TIZEN_SDK_ROOT/sdk.info"
+    ln -sf "$TIZEN_SDK_DATA_PATH/.tizen-cli-config" "$TIZEN_SDK_ROOT/tools/.tizen-cli-config"
+
+    # Use Tizen developer platform certificate as defaultls
+    cp "$TIZEN_SDK_ROOT"/tools/certificate-generator/certificates/distributor/sdk-platform/* \
+        "$TIZEN_SDK_ROOT"/tools/certificate-generator/certificates/distributor/
+
+    # Make symbolic links relative
+    find "$TIZEN_SDK_SYSROOT/usr/lib64" -maxdepth 1 -type l | while IFS= read -r LNK; do
+        ln -sf "$(basename "$(readlink "$LNK")")" "$LNK"
+    done
+
+    ln -sf ../../lib64/libcap.so.2 "$TIZEN_SDK_SYSROOT/usr/lib64/libcap.so"
+    ln -sf openssl3.pc "$TIZEN_SDK_SYSROOT/usr/lib64/pkgconfig/openssl.pc"
+
+    info "Done."
+    echo
+
+    # Information on necessary environment variables
+    warning "Before proceeding with Matter export environment variables as follows:"
```

Comment:
When installing arm and arch64, this section from arm installation will be "lost" in the output history. I would prefer to see instruction for user printed lastly.